src/Steeltoe.Tooling/steeltoe.rc/registry/dotnet.yml

```diff
@@ -0,0 +1,4 @@
+netcoreapp2.1: mcr.microsoft.com/dotnet/core/sdk:2.1
+netcoreapp2.2: mcr.microsoft.com/dotnet/core/sdk:2.2
```

Comment:
Should 2.2 be here?

Reply:
No.  I'll open a separate ticket and resolve post 0.7.1